beacon-chain/blockchain/receive_block.go

```diff
@@ -229,24 +240,32 @@ func (s *Service) handleDA(
 	block interfaces.SignedBeaconBlock,
 	blockRoot [32]byte,
 	avs das.AvailabilityStore,
-) (time.Duration, error) {
-	daStartTime := time.Now()
-	if avs != nil {
-		rob, err := blocks.NewROBlockWithRoot(block, blockRoot)
-		if err != nil {
-			return 0, err
-		}
-		if err := avs.IsDataAvailable(ctx, s.CurrentSlot(), rob); err != nil {
-			return 0, errors.Wrap(err, "could not validate blob data availability (AvailabilityStore.IsDataAvailable)")
+) (elapsed time.Duration, err error) {
+	defer func(start time.Time) {
+		elapsed := time.Since(start)
```

Comment:
Wondering about the scoping here. Does this method work as expected or is the returned value `elapsed` always 0?

This reads like you shadowing the return value `elapsed`. 

```suggestion
		elapsed = time.Since(start)
```

If I am right, pls add regression test!

Reply:
Fixed in 06eaaed0518f43a93b77e225f61328ece0d714bf.